po/Makevars

```diff
@@ -57,7 +57,7 @@ USE_MSGCTXT = no
 # Useful options are in particular:
 #   --previous            to keep previous msgids of translated messages,
 #   --quiet               to reduce the verbosity.
-MSGMERGE_OPTIONS = --add-location=never
+MSGMERGE_OPTIONS = --no-location
```

Comment:
Commit 55ffff9a5e0bd87fd4e8720a35b65befde9fe965 did the inverse of this, IIRC it was because of a difference between older and newer versions of msgmerge. It could be that adding both may work, I'll have to check.